scss/_variables.scss

```diff
@@ -6,12 +6,18 @@ $grouparoo-yellow: #ff7518;
 $grouparoo-light-yellow: #fff8f1;
 $grouparoo-red: #ff0039;
 $grouparoo-blue: #29abe2;
+$grouparoo-mint: #2fe5a4;
 $grouparoo-light-blue: #b3e8ff;
 $grouparoo-blue-gray: #9b9db1;
 $grouparoo-violet: #9954bb;
 $grouparoo-light-violet: #dfe8fb;
+$grouparoo-sky-blue: #6090ed;
+$grouparoo-spice: #edac60;
+$grouparoo-light-violet: #dfe8fb;
 $grouparoo-background-blue: #f6fafb;
 $grouparoo-green: #3fb618;
+$grouparoo-solid: #3d3d3d;
+$grouparoo-ethereal-blue: #f5f9ff;
 
 // Bootstrap Colors, cannot be CSS variables
 $primary: $grouparoo-blue;
```

Comment:
Curious why we're only declaring these.  I know at least "info" is also in use.  May want to audit the site to see if there are others!